src/less/RevisionPopupWidget.less

```diff
@@ -0,0 +1,29 @@
+.ext-wwt-revisionPopupWidget-attribution {
+	margin-top: 14px;
+}
+
+/* This matches the native MediaWiki CSS for diff sizes, which may not be available on article pages. */
+
+.mw-diff-bytes {
+	font-style: italic;
+
+	&::before {
+		content: '(';
+	}
+
+	&::after {
+		content: ')';
+	}
+}
+
+.mw-plusminus-pos {
```

Comment:
I was going to say that these colours could be used from `mediawiki.special.changeslist`, but that's got other stuff in it that we don't really need. But maybe a comment saying where they're defined in MediaWiki would be good.

Reply:
Just now saw this comment; I ended up using `mediawiki.special.changeslist`. It doesn't seem very big so hopefully okay?